workflow/automation/install_scripts/install_realisation.py

```diff
@@ -0,0 +1,176 @@
+import argparse
+from pathlib import Path
+
+from qcore import constants, simulation_structure, utils, qclogging
+
+
+def load_args():
+    parser = argparse.ArgumentParser(allow_abbrev=False)
+    parser.add_argument("cs_root", type=Path)
+    parser.add_argument("realisation")
+    parser.add_argument("--log_dir", type=Path)
+
+    vm_pert = parser.add_mutually_exclusive_group()
+    vm_pert.add_argument(
+        "--vm_perturbations",
+        action="store_true",
+        help="Use velocity model perturbations. If this is selected all realisations must have a perturbation file.",
+    )
+    vm_pert.add_argument(
+        "--ignore_vm_perturbations",
```

Comment:
Option name is slightly long, but I can accept this

Reply:
What happens both --vm_perturbations and --ignore_vm_perturnations are set? I can sense these two are not meant to be behave in opposite ways...or are they?

Reply:
On line 13 a mutually exclusive group is made, so argparse will prevent both options being chosen at the same time
The option name is inherited from the current install_cybershake.py script. But in this case, the script is not expected to be used by a user, only through the automated workflow so length of argument name is less important than clarity